cmd/analyzer/parse_args.go

```diff
@@ -122,18 +148,19 @@ func parseCmdLine(cmdlineArgs []string) error {
 func ParseInArgs(cmdlineArgs []string) (*InArgs, error) {
 	args := InArgs{}
 	flagset := flag.NewFlagSet("vpc-network-config-analyzer", flag.ContinueOnError)
-	args.InputConfigFile = flagset.String(InputConfigFile, "", "file path to input config")
-	args.InputSecondConfigFile = flagset.String(InputSecondConfigFile, "", "file path to the 2nd input config; "+
+	args.InputConfigFile = flagset.String(InputConfigFile, "", "Required. File path to input config")
+	args.InputSecondConfigFile = flagset.String(InputSecondConfigFile, "", "File path to the 2nd input config; "+
 		"relevant only for analysis-type diff_all_endpoints and for diff_all_subnets")
-	args.OutputFile = flagset.String(OutputFile, "", "file path to store results")
+	args.OutputFile = flagset.String(OutputFile, "", "File path to store results")
 	args.OutputFormat = flagset.String(OutputFormat, TEXTFormat,
-		"output format; must be one of "+getSupportedValuesString(supportedOutputFormats))
+		"Output format; must be one of:\n"+getSupportedValuesString(supportedOutputFormatsList))
```

Comment:
```suggestion
		"Output format; must be one of:\n"+strings.Join(supportedOutputFormatsList, ", "))
```
Also remove the function `getSupportedValuesString`